components/PageLink.tsx

```diff
@@ -2,7 +2,7 @@ import { FunctionComponent } from 'react';
 
 import styles from './PageLink.module.scss';
 import Icon from './Icon';
-import Link from 'next/link';
+import Link from '../components/Link';
```

Comment:
I _think_ this should do the trick @hugihlynsson, don't know of a way to reproduce on dev or Vercel. Shall we push to prod and see what happens?